plugins/generator-1.21.1/neoforge-1.21.1/mappings/eventparameters.yaml

```diff
@@ -0,0 +1,12 @@
+_default: null
+DAMAGE_AMOUNT:

```

Comment:
Since multiple events will potentially have damage amount altering, it may make sense to name this one more specifically, e.g., INCOMING_DAMAGE_AMOUNT